lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -214,7 +214,11 @@ def self.get_values(json, category_type, override_price = false)
       'i3.4xlarge' => 'Storage Optimized High I/O Quadruple Extra Large', 'i3.8xlarge' => 'Storage Optimized High I/O Extra Large',
       'i3.16xlarge' => 'Storage Optimized High I/O Hextuple Extra Large',
       'i3.metal' => 'Storage Optimized High I/O Metal',
-       'd2.xlarge' => 'Dense Storage Extra Large', 'd2.2xlarge' => 'Dense Storage Double Extra Large', 'd2.4xlarge' => 'Dense Storage Quadruple Extra Large', 'd2.8xlarge' => 'Dense Storage Eight Extra Large',
+      'i3p.16xlarge' => "Storage Optimized VMWare High I/O",
```

Comment:
VMWare -> VMware

Reply:
why do we get rid of Hextuple Extra Large in the name? (too long?)

Reply:
These are from a separate PR; merely merged the branch in to allow me to use the example i3p in testing.